nixos/modules/services/networking/iperf3.nix

```diff
@@ -5,10 +5,15 @@ let
   api = {
     enable = mkEnableOption (lib.mdDoc "iperf3 network throughput testing server");
     port = mkOption {
-      type        = types.ints.u16;
+      type        = types.port;
       default     = 5201;
       description = lib.mdDoc "Server port to listen on for iperf3 client requests.";
     };
+    multipleServerPorts = mkOption {
```

Comment:
I think it makes more sense then to have port renamed to ports, and have it take either a number or list of numbers? Having 2 ways of defining the same thing sounds weird.

Reply:
The idea behind this was to preserve backwards compatibility. Renaming `port` to `ports` and changing its type will break things.

Reply:
Maybe throw in a deprecation warning then? So that in the future we can reduce the size of this expression by removing the old one.